engine/src/contract_methods/connector/internal.rs

```diff
@@ -822,6 +829,16 @@ impl<I: IO + Copy> EthConnectorContract<I> {
         self.io.return_output(format!("\"{balance}\"").as_bytes());
     }
 
+    /// Return `nETH` balances for accounts (ETH on NEAR).
+    pub fn ft_balances_of(&mut self, accounts: Vec<AccountId>) {
+        let mut balances = aurora_engine_types::HashMap::new();
+        for account_id in accounts {
+            let balance = self.ft.ft_balance_of(&account_id);
+            balances.insert(account_id, balance);
+        }
+        self.io.return_output(&balances.try_to_vec().unwrap());
```

Comment:
Borsh has been bumped to 1.x.x.
```suggestion
        self.io.return_output(&borsh::to_vec(&balances).unwrap());
```